kfs/src/main.rs

```
@@ -0,0 +1,56 @@
+#![no_std]
+#![no_main]
+
+use log::log;
+
+#[panic_handler]
+fn panic_handler(info: &core::panic::PanicInfo) -> ! {
+    let _ = match info.location() {
+        Some(location) => log!("panic({location}): {}\n", info.message()),
+        None => log!("panic(location): {}\n", info.message()),
```

Might be better to say `panic(unknown location)` instead of `panic(location)`